src/dispatch/conference/models.py

```diff
@@ -11,6 +11,7 @@
 class Conference(Base, ResourceMixin):
     id = Column(Integer, primary_key=True)
     conference_id = Column(String)
+    conference_challenge = Column(String, nullable=False, server_default="N/A")
```

Comment:
I'm not sure there is a huge value to using `server_default` vs `default` here.

Reply:
When I had default, it didn't migrate the existing conference rows and set a default value which then broke the UI.